GVFS/GVFS/CommandLine/DiagnoseVerb.cs

```diff
@@ -156,6 +156,12 @@ protected override void Execute(GVFSEnlistment enlistment)
                             this.CopyFile(GVFSPlatform.Instance.GetDataRootForGVFS(), archiveFolderPath, LocalGVFSConfig.FileName);
                         }
 
+                        // kernel panics

```

Comment:
Nit: I don't think this comment is adding much, I think it can be removed.

Reply:
I can take it out.  We seem to do this pattern for most copies in the file which is why I did it here.

Reply:
> We seem to do this pattern for most copies in the file which is why I did it here.

Apologies if that's the case and I missed it.  If we're doing it elsewhere it's good with me to do it here too.

Reply:
@wilbaker, it honestly seemed redundant to me too.  I think it's fine to leave out.